proxy/src/main.rs

```diff
@@ -2,33 +2,53 @@ use proxy::coco;
 use proxy::env;
 use proxy::http;
 use proxy::registry;
+use structopt::{clap::AppSettings, StructOpt};
```

Comment:
The dependency graph of the project is alreayd sizable, do we really need to pull in a crate that depends on clap?

Reply:
I opted for `structopt` to have the proxy provide a nice help message. If we can do without then I’m happy to revert to `pico-args`.